src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -118,6 +118,7 @@ export default function FormNav(props) {
         // function to be called inadvertently
         if (
           formConfig.useCustomScrollAndFocus &&
+          page.scrollAndFocusTarget &&
```

Comment:
Do you need this new line for it to work? This is what `useCustomScrollAndFocus: true` is for, right?

Reply:
removed.